frontend-cert/css-projects/progress-bar/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Progress Bar</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="progress-bar-container">
+        <h2>Car Acceleration</h2>
+        <div class="progress-bar">
+            <span class="percentage"></span>
+        </div>
+        <div class="markings">
+            <span>0</span><span style="margin-left: 200px">100</span>
+            <span style="margin-left: 250px;">180</span>
+            <span style="margin-left: 260px;">300</span>
```

Comment:
I think it is best if we avoid mixing inline and external styles. We just keep everything inside the styles.css